packages/aws-cdk-lib/aws-eks/lib/alb-controller.ts

```diff
@@ -130,6 +170,198 @@ export enum AlbScheme {
   INTERNET_FACING = 'internet-facing'
 }
 
+/**
+ * Version of helm chart for Kubernetes for AWS ALB controller.
+ *
+ * @see https://github.com/aws/eks-charts/blob/v0.0.133/stable/aws-load-balancer-controller/Chart.yaml
+ */
+export enum HelmChartVersion {
+  /**
+   * v1.5.2
+   */
+  V1_5_2 = '1.5.2',
+
+  /**
+   * v1.5.1
+   */
+  V1_5_1 = '1.5.1',
+
+  /**
+   * v1.5.0
+   */
+  V1_5_0 = '1.5.0',
+
+  /**
+   * v1.4.8
+   */
+  V1_4_8 = '1.4.8',
+
+  /**
+   * v1.4.7
+   */
+  V1_4_7 = '1.4.7',
+
+  /**
+   * v1.4.6
+   */
+  V1_4_6 = '1.4.6',
+
+  /**
+   * v1.4.5
+   */
+  V1_4_5 = '1.4.5',
+
+  /**
+   * v1.4.4
+   */
+  V1_4_4 = '1.4.4',
+
+  /**
+   * v1.4.3
+   */
+  V1_4_3 = '1.4.3',
+
+  /**
+   * v1.4.2
+   */
+  V1_4_2 = '1.4.2',
+
+  /**
+   * v1.4.1
+   */
+  V1_4_1 = '1.4.1',
+
+  /**
+   * v1.4.0
+   */
+  V1_4_0 = '1.4.0',
```

Comment:
Anyone using these old versions of helm? I think we can get by with an enum-like class here too:

```typescript
class HelmChartVersion {
  public static V1_4_1 = new HelmChartVersion('1.4.1');
  public static V1_5_1 = new HelmChartVersion('1.5.1');

  public constructor(public version: string) {}
}
```

Let people just specify their other versions I think

Reply:
Good point. I opted against using an enum-like class here though. I'm wondering if this would be too much abstraction for what would essentially just be a string? Instead, I decided to hard code the helm chart version for the static alb controller versions that create instances of the alb controller class similar to the already hard coded alb controller version that is passed to the constructor.